### DISCUSSION TITLE
make the download and favorite buttons larger in the lazer web interface

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#20692] [Ideas] [answered]
currently the download and favorite buttons are quite small, so downloading multiple beatmaps can get tedious having to constantly exit the beatmap page because you accidently hit outside the download button.
This should be changed to just be either upper or lower half of the grey area (see image; top is current, bottom is proposed)

![osu_2022-10-10_15-07-20](https://user-images.githubusercontent.com/72625258/194937397-42924c6b-96b5-46df-a2ad-222cc135baa0.png)

**Comment by peppy** [comment]
I am fine with this change being made. Should be a pretty simple one. May want to add a rounded box when hovering to make it more obvious as to where the clickable region is.

**Comment by frenzibyte** [comment, marked as answer]
#20743
